lib/Macros/Sources/SwiftMacros/DistributedResolvableMacro.swift

```diff
@@ -133,6 +137,9 @@ extension DistributedResolvableMacro {
     var isGenericOverActorSystem = false
     var specificActorSystemRequirement: TypeSyntax?
 
+    let attributes = proto.attributes.filter { attr in
+      attr.as(AttributeSyntax.self)?.attributeName.trimmed.description == "_spi"
```

Comment:
@ahoppen would you mind helping me out what a better way to spell this is?

Reply:
I would suggest

```swift
    let attributes = proto.attributes.filter {
      $0.as(AttributeSyntax.self)?.attributeName.as(IdentifierTypeSyntax.self)?.name == .identifier("_spi")
    }
```

Also, you should be able to just do "\(attributes)" below and shouldn’t need `raw:` or `description` or `joined`

Reply:
Thank you, that's nicer! Weirdly enough the == .identifier() does not compare as true, but I could do `.text == "spi"` to work around that.

Also removed the raw: below, thanks!